### DISCUSSION TITLE
How to merge triangular faces into polygons?

### DISCUSSION THREAD

**Discussion by RikiLee** [CGAL/cgal#7938] [Q&A] [answered]
For example, a cube can be represented by 12 triangles or 6 squares. I want to use polygons to represent  a 3D mesh instead of triangles.

**Comment by sloriot** [comment, marked as answer]
This function might help: https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__meshing__grp.html#ga7fca6fa2db94560ab6d32e6a77fc35b6
